images/golang-tools/Dockerfile.arm64v8

```diff
@@ -0,0 +1,37 @@
+FROM multiarch/debian-debootstrap:arm64-stretch-slim
+ARG GO_VERSION
+
+# gcc for cgo
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		g++ \
+		gcc \
+		libc6-dev \
+		make \
+		pkg-config \
+        ca-certificates \
+        git \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p /usr/local/go \
+	&& wget -O go.tgz "https://golang.org/dl/go${GO_VERSION}.linux-arm64.tar.gz" \
+	&& tar -C /usr/local -xzf go.tgz \
+	&& rm go.tgz; 
+
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
+WORKDIR $GOPATH
+
+ENV GOLINT_PATH=$GOPATH/src/github.com/golang/lint
+RUN mkdir -p $GOLINT_PATH && cd $GOLINT_PATH && \
+    git clone https://github.com/golang/lint . && \
+    git reset --hard e14d9b0f1d332b1420c1ffa32562ad2dc84d645d && \
```

Comment:
maybe we should make these commits buildargs as well?

Reply:
good idea! I've put it to buildargs and also extracted part of install tooling so that it is done multiple times. Concluding we can now declaratively build different go versions for different archs including different tooling versions. This bloody thing is truly getting complex :D